### PR TITLE
Update cloud.md with guidelines that came out of the AI/ML Working Group

### PR DESCRIPTION
**Pull request by sirkitree**
This combined document incorporates the most up-to-date information from the SaaS Evaluation document while retaining relevant details from the current cloud.md document.

### CODE REVIEW

File: communications/cloud.md
Comment (deviantintegral) [inline communications/cloud.md:24]: 🤔 I wonder if we need this second line. Like some of these may not be "be cautious" but "do not use", but we don't actually have a documented list that way.

File: communications/cloud.md
Comment (deviantintegral) [inline communications/cloud.md:23]:
```suggestion
   - Is the company based in the United States or an allied country?
```

File: communications/cloud.md
Comment (deviantintegral) [inline communications/cloud.md:38]: Let's prioritize the one that is more broadly accessible first.

```suggestion
   - Check sources like Google News and Hacker News for reported incidents.
```

File: communications/cloud.md
Comment (deviantintegral) [inline communications/cloud.md:42]: Since this links to a PNG, how about we embed it directly?

File: communications/cloud.md
Comment (deviantintegral) [inline communications/cloud.md:79]: NotebookLM is the only one that is "approved" - and what I mean by that is, we don't have team accounts or anything like that for the other two.

I wonder if instead this could not be a section on "Approved services" but something like "Examples of Services Needing Evaluation by the Security Team"? And it should probably be broader than just AI too, and then go down after the Introducing New Services section.

**Review by deviantintegral — COMMENTED** [review]
Thanks so much for this. It's well on the way!

File: communications/cloud.md
Comment (sirkitree) [reply communications/cloud.md:24]: What do you think it would take to get this list? Or should we just remove the line?

File: communications/cloud.md
Comment (sirkitree) [reply communications/cloud.md:42]: Do you mean upload the file and embed or actually just embed in the .md file?

**Comment by deviantintegral** [comment]
@Lullabot/security I'd hoped to get another review of this in before the AI/ML meeting tomorrow... and clearly not. Can someone else pick this up in the next day or two and give it a read?

**Comment by bskowron2** [comment]
> @Lullabot/security I'd hoped to get another review of this in before the AI/ML meeting tomorrow... and clearly not. Can someone else pick this up in the next day or two and give it a read?

Hey there! Sorry for the slow response. I've read this and it looks great to me. Nice job!

**Review by deviantintegral — APPROVED** [review]

**Pull request merged**
